playgrounds/react/src/examples/form/components/form-context.ts

```diff
@@ -0,0 +1,12 @@
+import { createContext } from 'react';
+import { FormValidationResult, Rule } from './types';
+
+interface FormContextValue {
```

Comment:
```suggestion
/*
The validation-related props should move to the future validation context because the validation can be used separately from the form. Of course, the form should create validation context implicitly. 
*/
interface FormContextValue {
```